website/documentation/050-tutorials/content/howto/maintain-shoot/_index.md

```diff
@@ -9,21 +9,38 @@ scope: operator
+A version update during the maintenance time triggers a Shoot reconciliation.
 
-Since Kubernetes follows [Semantic Versioning](http://semver.org/), if indicated so, Gardener will automatically apply the patch release updates. But it will never auto update the Major or Minor releases since there is no effort to keep backward compatibility in those releases.
+{{% notice tip %}}
+<p>To manually trigger the maintenance operation, the Shoot can be annotated with `shoot.gardener.cloud/operation: maintain`.</p>
+{{% /notice %}}
 
-Major or Minor updates must be handled by updating the `.spec.kubernetes.version` field manually, these updates will be executed immediately and will not wait for maintenance time window. **Before applying such update on Minor or Major releases, operators should check for all the breaking changes introduced in the target release Changelog**.
+### Version Classifications
 
-E.g. If you have a Shoot cluster with the field values below (only related fields are shown):
+Gardener classifies versions into `preview`,`supported`, `deprecated` and `expired`.
+Please see [here for more information](https://github.com/gardener/gardener/blob/master/docs/usage/shoot_versions.md#version-classifications).
+Gardener takes version classifications into account during the maintenance operations as part of the efforts to ensure stable version upgrades without negatively impacting the workload deployed in the cluster.
+As such, `preview` versions are excluded from updates during the maintenance maintenance.
+This is because `preview` versions are typically recently releases version that have not yet undergone thorough testing and may contain bugs or security issues.
+
+### Automatic Kubernetes Version Upgrades
+
+If a Shoot has `.spec.maintenance.autoUpdate.kubernetesVersion: true` in the manifest, and the `.spec.kubernetes.versions` field in the CloudProfile used in the Shoot is updated, then the maintenance controller will apply Kubernetes [patch releases](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#patch-releases) updates automatically during the daily maintenance time window.
+
+Since Kubernetes follows [Semantic Versioning](http://semver.org/), if indicated so, Gardener automatically determines and applies the patch release updates. 
+It will never auto update the Major releases since there is no effort to keep backward compatibility in those releases. While this is also true for minor versions, (forceful minor version updates)[forceful-version-upgrades] can  happen during the maintenance time window (even though specifiying `.spec.maintenance.autoUpdate.kubernetesVersion: false`).
```

Comment:
```suggestion
It will never auto update the Major releases since there is no effort to keep backward compatibility in those releases. While this is also true for minor versions, (forceful minor version updates)[forceful-version-upgrades] can  happen during the maintenance time window (even though specifying `.spec.maintenance.autoUpdate.kubernetesVersion: false`).
```